saleor/graphql/schema.graphql

```diff
@@ -19124,6 +19142,25 @@ input CheckoutLineInput {
   price: PositiveDecimal
 }
 
+input CheckoutValidationRules {
+  """
+  The validation rules that can be applied to provided shipping address data.
+  """
+  shippingAddress: CheckoutAddressValidationRules
+}
+
+input CheckoutAddressValidationRules {
+  """
+  Determines if an error should be raised when the provided address doesn't have all required fields. Providing `country` code is always required.
```

Comment:
Determines if an error should be raised when the provided address doesn't have all the required fields. The list of required fields is dynamic and depends on the country code (use the `addressValidationRules` query to fetch them). Note: country code is mandatory for all addresses regardless of the rules provided in this input.